awx/main/tasks.py

```diff
@@ -1215,6 +1215,8 @@ def event_handler(self, event_data):
             else:
                 event_data['host_name'] = ''
                 event_data['host_id'] = ''
+            if event_data.get('event') == 'playbook_on_stats':
+                event_data['host_map'] = self.host_map
```

Comment:
Does this mean the API will start showing this (relatively large) mapping in the playbook_on_stats event data?

Reply:
This key doesn't get saved into the database - it only gets passed over the message bus (because it's not in `BasePlaybookEvent.VALID_KEYS`).

Reply:
https://github.com/ansible/awx/pull/6992/files#diff-9b1e5b80fcb01fab3e7a0ece4e371a50R410